sunpy/io/fits.py

```diff
@@ -40,6 +40,7 @@
 from astropy.io import fits
 
 from sunpy.io.header import FileHeader
+from sunpy.time import is_time, parse_time
```

Comment:
I found some errors when playing around with EIT wave stuff when writing out FITS files from the simulated data.  The time modules were imported to fix those problems.  That change must have crept in by mistake.  The imports must have crept in by mistake.  These imports are not used.